### PR TITLE
feat(audio): Add audio for each keypress

### PR DESCRIPTION
**Pull request by jhsu**
read out the letter pressed

### CODE REVIEW

File: src/game/game.ts [hunk 1]
```diff
@@ -116,6 +117,8 @@ export class Game extends EventEmitter<GameEvents> {
   }
 
   @action handleInput(key: string) {
+    console.log()
```
Comment (sifferhans) [inline src/game/game.ts]: Oops, seems like you left a console.log here :)

Comment (jhsu) [reply src/game/game.ts]: whoops! thanks for catching that

**Comment by danielgolden** [comment]
I've been using kdtype to review the alphabet in Spanish with my daughter. I love the idea, but this would add a little friction to that.

**Comment by jhsu** [comment]
sound was added, so this isn't needed

**Pull request closed without merging**